react/private/Icon/Icon.js

```diff
@@ -21,10 +34,14 @@ export default function Icon({ markup, className, svgClassName, ...restProps })
 Icon.propTypes = {
   markup: PropTypes.string.isRequired,
   svgClassName: PropTypes.string,
-  className: PropTypes.string
+  className: PropTypes.string,
+  size: PropTypes.string
```

Comment:
I wonder if we should provide a shared set of size prop types, so the the developer experience is a bit nicer 🤔 

Don't have to do this right away, since we don't have a good prop types experience today with the boolean props.

Reply:
Great pickup, do we have a centralised place to have prop types descriptions in styleguide? 🤔 Or I can just put it in the text component / decorator and refer from there

Reply:
Exported from to decorator 👍